.github/workflows/msbuild.yml

```diff
@@ -19,6 +17,8 @@ jobs:
       fail-fast: false
     steps:
     - uses: nuget/setup-nuget@v1
+      with:
+        nuget-version: latest
```

Comment:
This should already be latest https://github.com/NuGet/setup-nuget/blob/ba2372f3807bc5ec02a67dfb96cd7c5d6b72a6c4/action.yml#L5-L7